src/azure-cli/azure/cli/command_modules/servicefabric/operations/managed_node_types.py

```diff
@@ -0,0 +1,273 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long,too-many-lines
+
+from azure.cli.core.commands import LongRunningOperation
+from azure.cli.command_modules.servicefabric._sf_utils import (_log_error_exception)
+from azure.mgmt.servicefabric.models import (ErrorModelException,
+                                             NodeType,
+                                             EndpointRangeDescription,
+                                             VMSSExtension,
+                                             VaultSecretGroup,
+                                             VaultCertificate,
+                                             SubResource)
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def create_node_type(cmd,
+                     client,
+                     resource_group_name,
+                     cluster_name,
+                     node_type_name,
+                     instance_count,
+                     primary=False,
+                     disk_size=None,
+                     application_start_port=None,
+                     application_end_port=None,
+                     ephemeral_start_port=None,
+                     ephemeral_end_port=None,
+                     vm_size=None,
+                     vm_image_publisher=None,
+                     vm_image_offer=None,
+                     vm_image_sku=None,
+                     vm_image_version=None,
+                     capacity=None,
+                     placement_property=None):
+
+    #  set defult parameters
+    if disk_size is None:
+        disk_size = 100
+
+    if vm_size is None:
+        vm_size = "Standard_D2"
```

Comment:
Pay attention to case. Maybe case insensitive is better.

Reply:
doing it the same way they have it in the examples of the sdk documentation. it's working correctly